ui/hooks/src/use-valid-did.ts

```diff
@@ -16,11 +14,8 @@ export function useValidDid(profileId: string, enabled?: boolean) {
     if (enabled) {
       setIsLoading(true);
       resolveDid(profileId).then(res => {
-        // case when it's an eth address and we need to verify if it's a valid address separately
-        if (res && profileId.includes('eip155')) {
-          ethers.utils.isAddress(profileId.split(':').pop())
-            ? setValidDid(true)
-            : setValidDid(false);
+        if (res && 'isEthAddress' in res) {
```

Comment:
check this https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwn